src/index.js

```diff
@@ -1,2 +1,25 @@
 import './style.css';
-import './modules/addScore.js';
+import * as API from './modules/LeaderboardAPI.js';
+
+const refresh = document.querySelector('.recent-scores-refresh-button');
+const form = document.querySelector('.add-your-scores-container');
+
+document.addEventListener('DOMContentLoaded', API.showScores);
+
+form.addEventListener('submit', (e) => {
+  console.log('are you working?');
```

Comment:
[OPTIONAL] Your code is working great, good job! It would be even better if you removed the unnecessary console logs.

Reply:
@smunozmo thank you for that feedback. console.log has been removed 
![e15](https://user-images.githubusercontent.com/40554384/162550870-258d2888-1d47-4806-8333-ddca3aee0b45.PNG)

Reply:
Looks perfect! 💯